<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7ISFMHR" ignored="0" name="Default test" tests="6" failures="1" timestamp="2023-04-22T02:34:32 IST" time="116.893" errors="0">
  <testcase name="m_login" time="14.593" classname="com.selenium.tests.TC_001_SystemTest"/>
  <testcase name="m_RegisterPatient" time="23.322" classname="com.selenium.tests.TC_001_SystemTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Male] but found [[Ljava.lang.String;@5ff60a8c],
	expected [Test,Calab] but found [[Ljava.lang.String;@67b7c170],
	expected [Test,Main Street,Newyork,NY,USA,10022] but found [[Ljava.lang.String;@889d9e8],
	expected [10,February,2000] but found [[Ljava.lang.String;@ba354ca],
	expected [12345678] but found [[Ljava.lang.String;@acf859d]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Male] but found [[Ljava.lang.String;@5ff60a8c],
expected [Test,Calab] but found [[Ljava.lang.String;@67b7c170],
expected [Test,Main Street,Newyork,NY,USA,10022] but found [[Ljava.lang.String;@889d9e8],
expected [10,February,2000] but found [[Ljava.lang.String;@ba354ca],
expected [12345678] but found [[Ljava.lang.String;@acf859d]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.selenium.tests.TC_001_SystemTest.m_RegisterPatient(TC_001_SystemTest.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- m_RegisterPatient -->
  <testcase name="m_CheckPatientDetailpage" time="13.107" classname="com.selenium.tests.TC_001_SystemTest"/>
  <testcase name="m_Visit" time="1.451" classname="com.selenium.tests.TC_001_SystemTest"/>
  <testcase name="m_VerifyAttachment" time="20.082" classname="com.selenium.tests.TC_001_SystemTest"/>
  <testcase name="m_EndVisit" time="14.367" classname="com.selenium.tests.TC_001_SystemTest"/>
</testsuite> <!-- Default test -->
